### ISSUE TITLE
Unable to load trained tensorflow model to ML backend

### ISSUE THREAD

**Issue by PragathiJ** [heartexlabs/label-studio#621]
Hi, I am fairly new to label studio and I am using the tool for auto labeling the time series data. I am trying to load a pre-trained tensorflow model to the ML backend and passing the csv files that are loaded as tasks to the label studio as test data to the model inference. Although ML backend starts without any error, it does not work. The model is not getting connected with a message that says the model is not trained.

I am using keras load model function under init function: 
**self.model = keras.models.load_model('http://localhost:8000/model.h5')** and using **self.model.predict** under predict function of the class.

Could anyone please let me know if it supports tensorflow and if there is a possibility to load any pre-trained model to ML backend.

**Comment by srv-gabriel** [comment]
Hi,
I'm having the same error. Were you able to run the [example](https://labelstud.io/tutorials/dummy_model.html) correctly?

**Comment by makseq** [comment]
> The model is not getting connected with a message that says the model is not trained.

@PragathiJ Hi, could you provide any screenshots with this error?

**Comment by makseq** [comment]
Our ML docs and examples were corrected, hope they work as expected now.
